UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/FileShareServiceController.cs

```diff
@@ -16,5 +32,29 @@ public IActionResult CreateBatch(string correlationId)
             }
             return BadRequest();
         }
+
+        [HttpGet]
+        [Route("/batch")]
+        public IActionResult GetBatches([FromQuery] int? limit, [FromQuery] int start = 0, [FromQuery(Name = "$filter")] string filter = "")
+        {
+            if (limit != null && !string.IsNullOrEmpty(filter))
+            {
+                var response = fileShareService.GetBatches(filter);
+                if (response != null)
+                {
+                    return Ok(response);
+                }
+            }
+            return BadRequest();
+        }
+
+        [HttpGet]
+        [Route("/batch/{batchId}/files/{filesName}")]
+        public FileResult DownloadENCFile(string filesName)
+        {
+            byte[] bytes = fileShareService.GetEncFileData(filesName);
```

Comment:
Code to download readme.txt file has to be added and also change name of method DownloadFile()

Reply:
Did changes accordingly. Thanks!